genie/database_to_staging.py

```diff
@@ -35,6 +35,74 @@
 SV_CENTER_PATH = os.path.join(GENIE_RELEASE_DIR, "data_sv_%s.txt")
 BED_DIFFS_SEQASSAY_PATH = os.path.join(GENIE_RELEASE_DIR, "diff_%s.csv")
 
+FULL_MAF_RELEASE_SCHEMA = {
```

Comment:
One outstanding feature to this PR is if should we have a function that enforces these expected data types on the release file?

Reply:
Hmm, can you speak to how we would enforce these data types?  We do read the maf dataframe in with these dtypes right?

Reply:
Yeah it seems like the maf dataframe gets read in and it takes these dtypes. We don't have code in the read_csv that specifies the dtype by column, it's just a regular [read_csv](https://github.com/Sage-Bionetworks/Genie/blob/develop/genie/database_to_staging.py#L769-L771)

We could enforce it when we read it in like this for example: `df = pd.read_csv(maf_data, dtype=FULL_MAF_RELEASE_SCHEMA)`